exercises/word-count/instructions.md

```diff
@@ -0,0 +1,71 @@
+# Instructions
+
+Your task is to count how many times each word occurs in a subtitle of a drama.
+
+The dramas that the subtitles are taken from use only ASCII characters.
+
+The characters often speak in casual English, using contractions like _they're_ or _it's_.
+Though these contractions come from two words (e.g. _we are_), the contraction (_we're_) is considered a single word.
+
+Words can be separated by any form of punctuation (e.g. ":", "!", or "?") or whitespace (e.g. "\t", "\n", or " ").
+The only punctuation that does not separate words is the apostrophe in contractions.
+
+Numbers are considered words.
+So if the subtitles say `It costs 100 dollars.` then `100` will be its own word.
+
+Words are case insensitive.
+For example, the word `you` occurs three times in the following sentence:
+
+> You come back, you hear me? DO YOU HEAR ME?
+
+The ordering of the word counts in the results doesn't matter.
+
+The two following groups of counts (for the sentence `I scream, you scream, we all scream for ice cream`) are equivalent:
```

Comment:
I think you can eliminate this example, as it's going to make the instructions very long and I think it doesn't add huge value. The second example below is more useful I think.